eth-brownie/brownie
# chore: update .pre-commit-config.yaml

### What I did

Updated the pre-commit hooks.

### How I did it

1. Looked through the hooks and checked which ones are outdated and replaced with the latest versions.
2. run `pre-commit run --all`

### How to verify it

```sh
$ pre-commit run --all
```
**Note:** I've added a hook that eliminates trailing whitespaces and Windows specific End-Of-File encoding. This is a standard sanity check. More details [here](https://github.com/pre-commit/pre-commit-hooks#trailing-whitespace).
